components/BusinessSelectionModalV2/index.tsx

```diff
@@ -291,150 +291,92 @@ const BusinessSelectionModalV2 = ({ open }: BusinessSelectionModalProps) => {
                   return (
                     <>
                       <Form>
-                        {/* Anon user screen*/}
-                        {existingBusinesses.length === 0 && (
-                          <>
-                            {!showAdditionalConfiguration ? (
-                              <>
-                                {/* quick start  */}
-                                <h1 className="text-xl font-semibold mb-8">
-                                  Start the process by creating a business
-                                </h1>
-                                <FormikTextInput
-                                  updateContainerCss={basicCssClassUpdater(
-                                    'mb-4',
-                                  )}
-                                  fieldName="email"
-                                  label={'Email'}
-                                  type="email"
-                                  placeholder={'Enter email'}
-                                />
-
-                                <Button
-                                  fullWidth
-                                  disabled={!isValid || isSubmitting}
-                                  variant="contained"
-                                  color="primary"
-                                  onClick={() => handleSubmit()}
-                                >
-                                  Start new demo
-                                </Button>
-
-                                <p
-                                  onClick={() => {
-                                    setShowAdditionalConfiguration(true)
-                                    setSelectedUser(null)
-                                  }}
-                                  className="text-left text-sm hover:cursor-pointer text-[#1976d2] mt-4"
-                                >
-                                  Show additional configurations →
-                                </p>
-                              </>
-                            ) : (
-                              <AdditonalConfigOptions
-                                setShowAdditionalConfiguration={
-                                  setShowAdditionalConfiguration
-                                }
-                                handleSubmit={handleSubmit}
-                                isDisabled={isValid}
-                                platformEnabledProducts={
-                                  platform?.enabledProducts ??
-                                  Object.values(ProductType)
+                        <>
```

Comment:
I don't think we are doing "anon" any more

there was a bug in how the disabled state was being handled so I just deleted it.